client/src/Pages/MentorsView.js

```diff
@@ -65,14 +64,21 @@ const [mentorName, setMentorName] = useState(null);
           <ul className="student-list">
             {studentList.map(({ user_id, first_name, last_name }) => {
               return (
-                <li key={user_id} className="students-name ">
+                <li key={user_id} className="students-name">
                   <Link
+                    onClick={() => setBg({ activeIndex: user_id })}
                     to={`./MentorsView?studentId=${user_id}`}
-                    className="name-list"
+                    className="name-list d-flex"
                   >
-                    <div>{`${first_name} ${last_name}`}</div>
-                    <div>
-                      <i class="fas fa-arrow-right"></i>
+                    <div
+                      className={`${
+                        user_id == bg.activeIndex ? "active" : "inactive"
```

Comment:
May I suggest instead of adding another piece of state (`bg`) that we use the existing `studentId` from line 40 https://github.com/CodeYourFuture/knowledge-checklist/pull/14/files#diff-748a7bf14795541058c7fcc4dab14eaabe1da8b0e24d9d07fb971498b608f3d2L40
```suggestion
                        user_id == studentId ? "active" : "inactive"
```